src/handlers/proposalsHandlers.ts

```diff
@@ -22,10 +22,15 @@ import {
 import type { Request, Response, NextFunction } from 'express';
 
 const getProposals = (
-	req: Request,
+	req: AuthenticatedRequest,
 	res: Response,
 	next: NextFunction,
 ): void => {
+	if (req.user === undefined) {
+		next(new FailedMiddlewareError('Unexpected lack of user context.'));
```

Comment:
Tell me about it. _[shakes head]_ You and me both pal. _[raspberries in exasperation]_ What a day amiright

Reply:
`Unexpected lack of joke.`